backend/api/services/authorServices.py

```diff
@@ -22,6 +28,48 @@ def getAuthorJsonById(id):
     serializer = AuthorSerializer(a)
     return Response(serializer.data)
 
+# Checks the author table for a author with a specific URL
+def getRemoteAuthor(request, remote_author_create_func):
+    requestor_url = request.headers.get('X-Request-User')
+    #author = Author.objects.get(url=requestor_url)
+    author = Author.objects.filter(url=requestor_url)
+    if author.exists():
+        return author[0]
+    # At this point the author doesnt exist so we must create
+    else:
+        author = createRemoteAuthor(request, remote_author_create_func)
```

Comment:
I think we should get the author and then return it to the frontend. Saving it to own backend is redundant in my opinion

Reply:
It saves us a lot of requests in my opinion, which is a good performance boost on the heroku seervers. Cuz asking for the user on any like or comment interaction is a 4 msg process otherwise whereas if we store if becomes only 2 effectively doubling performance. The overhead of a connecting user is not too high.
Second reason is that it is just way easier to treat them as a local user lool, There is a lot less code to be written this way

Reply:
What if remote server `delete` or `edit` the user after it is being stored at your end. It is synchronize database rather than services interactions. To make sure data is all matched, communications between server is unavoidable

Reply:
As comment structure stands, I looked into it a bit more, to store a comment we must reference an author so itll be a bit hard. I could add an extra call that is called to update the user, but I also kind of dont wanna do a structure overhaul of something that is working